runner/ansible/roles/2.1.6/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.6'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    #   if rsc_colocation constraint is not configured, then fail
+    cibadmin -Q --xpath "//constraints/rsc_colocation" || exit 1
```

Comment:
Multiple times repeated strings here too `//constraints/rsc_colocation` and `//constraints/rsc_order`

Reply:
Agreed, could be set as shell var, but I couldn't test the result.

Reply:
We could have a couple of variables for these 2 xpath items.
But well, not a bid deal